exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -62,6 +62,7 @@ lazy_static! {
 }
 
 #[test]
+#[ignore]
```

Comment:
Are these the only ignores we have? Is it possible to add a comment to these annotations, e.g., `#[ignore="ECR-XYZ"]`?

Reply:
Would you add please a concise comment then explaining why they are ignored? Because we already have some ignores, e.g., in node_context.